dashboards/cora/cora.yaml

```diff
@@ -0,0 +1,6800 @@
+                ElementType: FILTER_CONTROL
+                RowIndex: 0
+                RowSpan: 1
+              - ColumnIndex: 6
+                ColumnSpan: 2
+                ElementId: 46abbf0e-787e-4ed4-ad67-61d3bbdc8035
+                ElementType: FILTER_CONTROL
+                RowIndex: 0
+                RowSpan: 1
+              - ColumnIndex: 8
+                ColumnSpan: 2
+                ElementId: 5bddc540-b508-4348-8345-03bb14c00ad8
+                ElementType: FILTER_CONTROL
+                RowIndex: 0
+                RowSpan: 1
+              - ColumnIndex: 10
+                ColumnSpan: 2
+                ElementId: eb9d8dbe-2508-42d6-9def-1994d976efd8
+                ElementType: FILTER_CONTROL
+                RowIndex: 0
+                RowSpan: 1
+              - ColumnIndex: 0
+                ColumnSpan: 2
+                ElementId: 07d358cb-9c91-4a09-8a16-991bd03b6c1c
+                ElementType: FILTER_CONTROL
+                RowIndex: 1
+                RowSpan: 1
+              - ColumnIndex: 2
+                ColumnSpan: 2
+                ElementId: 354bcc22-7d7d-467d-aeab-1e85019cd498
+                ElementType: PARAMETER_CONTROL
+                RowIndex: 1
+                RowSpan: 1
+              - ColumnSpan: 2
+                ElementId: 3d11854b-bd24-4d13-b098-13e59ff32b42
+                ElementType: PARAMETER_CONTROL
+                RowSpan: 1
+              - ColumnSpan: 2
+                ElementId: 638831ee-272a-4273-bb57-1af1cf3b5ee5
+                ElementType: PARAMETER_CONTROL
+                RowSpan: 1
+        SheetId: fcbb9302-e094-4049-b344-831c8e36007d
+        TextBoxes:
+        - Content: |-
+            <text-box>
+              <inline font-size="40px">Recommendations Table</inline>
+              <br/>
+              <inline font-size="14px">Use this table for Excel export</inline>
+              <br/>
+            </text-box>
+          SheetTextBoxId: 1531c453-a35b-4c63-aaee-62fd91ec588e
+        - Content: "<text-box>\n  <inline font-size=\"36px\">Recommendations Summary\_\_\
+            </inline>\n</text-box>"
+          SheetTextBoxId: c11a9de6-dd3c-4994-9a1e-b86bfc1ce458
+        - Content: |-
+            <text-box>
+              <br/>
+              <inline font-size="40px"/>
+              <inline font-size="40px">Scatter Plot</inline>
+            </text-box>
+          SheetTextBoxId: 1a5e9b89-f55b-4ed5-82f7-1b8ffd1e02f1
+        - Content: "<text-box>\n  <ul>\n    <li>\n      We recommend reviewing rightsizing\
+            \ first and then proceed with commitments. See\n      <a href=\"https://catalog.workshops.aws/awscid/en-US/dashboards/additional/cora#usage-guide\"\
+            \ target=\"_blank\">more</a>\n      .\n    </li>\n    <li>\n      Please note\
+            \ that there migt be a\n      <a href=\"https://catalog.workshops.aws/awscid/en-US/dashboards/additional/cora#difference-with-ui\"\
```

Comment:
Typo. Change to might and add space between a and difference

Reply:
```suggestion
            \ that there might be a&nbsp;\n      <a href=\"https://catalog.workshops.aws/awscid/en-US/dashboards/additional/cora#difference-with-ui\"\
```